functions/check-expiry/v2.js

```diff
@@ -9,6 +9,9 @@ const v2 = {
     // change the check status
     const checkData = await expireChecks(logger)
 
+    const checkCodes = checkData.map(e => e.checkCode)
+    logger(`check-expiry: Updated ${checkData.length} checks' status to \`EXP\`:\n${checkCodes.join('\n')}`)
```

Comment:
This will output 500 check codes.   Why?  It might be better to log each check expiry in the pupilEvent table then at least we can search for it.

Please note that Azure Application Insights will only sample data from functions - so not every logger output actually gets saved.